docs/running.md

```diff
@@ -5,9 +5,9 @@ The examples below are for Open MPI. Check your MPI documentation for arguments
 Typically one GPU will be allocated per process, so if a server has 4 GPUs, you would run 4 processes. In Open MPI,
 the number of processes is specified with the `-np` flag.
 
-Starting with the Open MPI 3, it's important to add the `-bind-to none` and `-oversubscribe` arguments. `-bind-to none`
-specifies Open MPI to not bind a training process to a single CPU core (which would hurt performance). `-oversubscribe`
-enables you to to run multiple training processes on a single core.
+Starting with the Open MPI 3, it's important to add the `-bind-to none` and `-map-by slot` arguments. `-bind-to none`
+specifies Open MPI to not bind a training process to a single CPU core (which would hurt performance). `-map-by slot`
+allows to have a mixture of different NUMA configurations since the default is to bind to the socket.
```

Comment:
allows --> allows you 
since --> because
default --> default behavior # or replace behavior with a more appropriate word